plugins/fclrave/fclspace.h

```diff
@@ -111,13 +137,17 @@ class FCLSpace : public boost::enable_shared_from_this<FCLSpace>
                     (*itgeompair).second.reset();
                 }
                 vgeoms.resize(0);
+
+                // make sure to clear vgeominfos after vgeoms because the CollisionObject inside each vgeom element has a corresponding vgeominfo as a void pointer.
+                vgeominfos.resize(0);
             }
 
             KinBody::LinkPtr GetLink() {
                 return _plink.lock();
             }
 
             KinBody::LinkWeakPtr _plink;
+            vector< boost::shared_ptr<GeometryInfo> > vgeominfos; ///< info for every geometry of the kinbody
```

Comment:
``info for every geometry of the kinbody`` may be a bit misleading, maybe ``info for every geometry of the link`` is more accurate?